app/helpers/analysis-status.js

```diff
@@ -6,6 +6,7 @@ export function analysisStatus(params) {
   if(typeof risk === "object") {
     risk = risk.value;
   }
+  risk = parseInt(risk);
```

Comment:
is the parseInt required?

Reply:
Yes, since the status is string and we are checking for int values